packages/nextlove/src/generators/generate-openapi/index.ts

```diff
@@ -139,9 +139,13 @@ export async function generateOpenAPI(opts: GenerateOpenAPIOpts) {
     file_path,
     { setupParams, routeSpec, route: routePath },
   ] of filepathToRouteFn) {
-    const isPostOrPutOrPatch = ["POST", "PUT", "PATCH"].some((method) =>
-      routeSpec.methods.includes(method)
-    )
+    const isGet = routeSpec.methods.includes("GET")
+
+    const isPostOrPutOrPatch =
+      !isGet &&
+      ["POST", "PUT", "PATCH"].some((method) =>
+        routeSpec.methods.includes(method)
+      )
```

Comment:
I feel like we should either 1) flag this or 2) have seam connect perform the filtering

Reply:
i'm leaning towards (2), I don't think nextlove should have this kind of preference

Reply:
Right now it does have a preference, so this update just switches the preference. We can flag it. We can't do 2 unless we update this function to generate all methods. Good first issue?

Reply:
I would like to merge this and update seam connect to use it. I can add a flag if we think there is impact other than seam connect, but I consider this a bug at the moment.